<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>global_reference_search_v_0_1_by_Yves</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_search_setting" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_select_setting1" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="margin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="label_select_folder">
            <property name="text">
             <string>选择文件夹：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_folder_path"/>
          </item>
          <item>
           <widget class="QPushButton" name="button_select_folder">
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_select_setting2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1,0,0,0">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="margin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="label_search_content">
            <property name="text">
             <string>搜索内容：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_search_content"/>
          </item>
          <item>
           <widget class="QLabel" name="label_search_file_format">
            <property name="text">
             <string>搜索文件格式：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_search_file_format"/>
          </item>
          <item>
           <widget class="QLabel" name="label_browse_line">
            <property name="text">
             <string>上下文预览行数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_browse_line"/>
          </item>
          <item>
           <widget class="QPushButton" name="button_search">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_search_result" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QLabel" name="label_search_result">
         <property name="text">
          <string>搜索结果：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget_search_result">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
